src/Chapter1.hs

```diff
@@ -429,9 +430,9 @@ task is to specify the type of this function.
 49
 -}
 
+squareSum :: Num a => a -> a -> a
```

Comment:
We explain `Num` in the following chapters, so you are one step ahead 😸 👏🏼 

In here `Int -> Int -> Int` would also work, as a more specific type 🙂

Reply:
yea, briefly read the learn haskell for great good book before starting this
also having the language server set up properly makes it easier :P